Fretboard.h

```diff
@@ -0,0 +1,33 @@
+/******************************************************************
+    Fretboard.h
+
+    structure to encapsulate a guitar fretboard
+******************************************************************/
+#pragma once
+#include <iostream>
+#include "LeakedObjectDetector.h"
+#include "Defines.h"
+#include "GuitarString.h"
+#include "Chord.h"
+#include "Fretboard.h"
```

Comment:
you are including Fretboard.h **in** Fretboard.h.
that doesn't make sense.
- [ ] revise

Reply:
copy paste error fixed.